src/TrackerCouncil.Smz3.Tracking/VoiceCommands/CheatsModule.cs

```diff
@@ -207,6 +207,15 @@ private GrammarBuilder SetupCrystalFlashRule()
             .OneOf("setup crystal flash requirements", "ready a crystal flash");
     }
 
+    [SupportedOSPlatform("windows")]
+    private GrammarBuilder ChargeShinesparkRule()
+    {
+        return new GrammarBuilder()
+            .Append("Hey tracker,")
+            .Optional("please", "would you kindly")
+            .OneOf("charge a shine spark", "ready a spine shark");
```

Comment:
After the fact: maybe we should add the lines that Betus tried? "Give me (a) shine spark", "Activate shine spark" and "Enable shine spark". Maybe also "turn on shine spark".

Reply:
Good idea. Let me put something together.

Reply:
Based on the recent discussion about uses, maybe you could add something like "I don't want to have to IBJ". lol

Reply:
Chaos mode where Tracker is listening for a bunch of phrases that don't start with "Hey Tracker"